### DISCUSSION TITLE
Span files not shrinking after retention settings

### DISCUSSION THREAD

**Discussion by rjl61** [datalust/seq-tickets#1444] [Help]
I have 2 seq instances with the secondary instance ingesting messages from the primary instance.

Retention settings are set identically on both systems.

The span files on the secondary remain at large sizes (eg  > 140MB each) when compared to those on the primary (< 100Kb) , which leads to disk shortages.

Not sure if it is related, but the Diagnostics report shows IsServerGC: No when it is Yes on the primary.

**Comment by nblumhardt** [comment]
Hi Richard,

If you're able to shoot screenshots of Data > Storage, along with diagnostic reports, through to our support address (`support@`) we might be able to spot something.

If you're replicating events for availability/DR purposes, some native capabilities are now included in Seq itself (including load balancer support), which will be a lot more reliable than the replication app. Let me know if you need some more info on this!

Best regards,
Nick

**Reply by rjl61** [reply]
Hi Nick,
Email sent as requested.
Yes please for the new capability information.
Thanks
Richard
